src/transformers/modeling_utils.py

```diff
@@ -2962,6 +3012,11 @@ def from_pretrained(
                 kwargs["skip_keys"] = model._skip_keys_device_placement
             dispatch_model(model, **kwargs)
 
+        if quantization_method_from_args == QuantizationMethod.GPTQ:
+            tokenizer = AutoTokenizer.from_pretrained(pretrained_model_name_or_path, use_fast=True)
```

Comment:
Here it would be nice to tell users what is going under the hood (a logger.info maybe saying that it is running quantization on xxx dataset)

Reply:
Actually there is already a logger.info on that method under the hood, feel free to ignore my comment

Reply:
I may also add a tqdm bar to show the progression

Reply:
I see, I would say yes let's maybe set `verbose=True` in that case to give more context to users!